index.html

```diff
@@ -0,0 +1,251 @@
+                             and understanding among science 
+                             technology, humanities, and the arts.
+                        </p>
+                    </div>
+                </li>
+        
+
+                <li class="speaker-box">
+                    <div class=" speaker-box_img"><img src="./assets/images/cvimage2.JPG" alt="speaker-image"></div>
+                    <div class="speaker-box_content">
+                        <h2 class="speaker-name">Barack Obama</h2>
+                        <p class="speaker-background">
+                            Berkman Professor of Entrepreneurial 
+                            Legal Studies at Harvard Law School
+                        </p>
+                        <p class="speaker-description">
+                            As the main venue for new media art production in Korea,
+                             Nabi promotes cross-disciplinary collaboration 
+                             and understanding among science 
+                             technology, humanities, and the arts.
+                        </p>
+                    </div>
+                </li>
+            </ul>
+            <ul class="speakers" id="WebDevSpeakers">
+                
+            </ul>
+            <article class="view-more" id="view-more-holder">
+                <button class="view-more-container" id="view-more-btn">
+                    <h4 id="clickformore">VIEW MORE</h4>
+                    <span id="dropformore" class="chevron-down"> <i class='bx bx-chevron-down'></i> </span>
+                </button>
+            </article>
+        </article>
+    </section>
+
+    <section class="footer-container">
+        <footer class="footer">
+            <article class="footer-top-container">
+                <article class="container footer-top">
+                    <h2 class="h-title">
+                        Our Partners
+                        <a href="#" class="text-border"></a>
+
+                    </h2>
+                    <ul class="partnerr-logo">
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/googlekorea 1.svg" alt="google"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/daumkakaologo 1.svg" alt="microsoft"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/airbnb 1.svg" alt="Bolt"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/mozilla 1.svg" alt="Mozilla"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/airbnb 1.svg" alt="google"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/googlekorea 1.svg" alt="microsoft"></span></li>
```

Comment:
- Good job done so far, but please it is important to stick to the layout in the original design, you have added too many sponsors in your sponsor section. I suggest you style and design this section accordingly. Please fix this 😄 
Your project:
<img width="591" alt="erreev" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/4f7abef7-13c0-4345-9867-4407a46e418a">

Design template:
<img width="629" alt="wefefwf" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/e8a65a11-3d9e-4cd6-b2ed-59ab63f632a4">